app/src/main/res/layout/fragment_dashboard.xml

```diff
@@ -23,4 +23,4 @@ tools:context=".DashboardActivity">
     android:layout_centerInParent="true"
     android:indeterminate="true"/>
 
-</RelativeLayout>
\ No newline at end of file
+</RelativeLayout>
```

Comment:
sorry I didn't see this before - `RelativeLayout` is deprecated because it has poor performance. Please use a different layout or combinations of layouts

Reply:
done